dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/frameworkregistryimports/FrameworkRegistryImportsUpdater.kt

```diff
@@ -17,18 +19,34 @@ class FrameworkRegistryImportsUpdater {
 
         val allRegisteredFrameworks = pathToFrameworkDirectory.toFile().listFiles {
                 file ->
-            file.isDirectory
+            file.isDirectory && file.listFiles()?.any { it.name == "BaseFrameworkDefinition.ts" } ?: false
```

Comment:
Why do we need this?

Reply:
When you switch between branches, git sometimes leaves empty directories lying around. I added this check to enforce that those empty directories do not get considered when generating the framework registry.

Reply:
ok, thank you